prov/sockets/configure.m4

```diff
@@ -23,7 +31,7 @@ AC_DEFUN([FI_SOCKETS_CONFIGURE],[
 				[],
 				[],
 				[sockets_shm_happy=1],
-				[sockets_shm_happy=0])
+				[sockets_shm_happy=0])])
```

Comment:
That was added to match the `AS_IF` that was added in this patch.  Looking at it again, it would have been better to add it to the next line instead to make the diff clearer.